### PR TITLE
Added contributions and bug reporting sections to README

### PR DESCRIPTION
**Pull request by greer184**
Add two sections to the README to facilitate issue reporting and feature development external to the Lotus Locke team. If anything makes sense to add to the descriptions (or my English is subpar), make notes of it and I'll update the PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,3 +18,28 @@ cargo run "127.0.0.1:8091" "127.0.0.1.8090"
...
+
+# Contributions
+
+To contribute to the project, either open a new issue for a bug or feature that currently is not
+listed there or find a bug or feature present in the issue list that you would like to address. 
+For features, simply provide a brief description of the feature and any design details that add
+clarity to the work that needs to be done for the feature to be considered "complete". 
+
+When setting up a Pull Request, remember to reference the issue number in the request so we know
+what to look for in review. Make sure the code is clean and free of extraneous comments or notes.
```
Comment (portends) [inline README.md]:
```suggestion
what to look for in a review. Make sure the code is clean and free of extraneous comments or notes.
```

**Review by portends — CHANGES_REQUESTED** [review]
small wording correction

File: README.md (hunk 1 above)
Comment (greer184) [reply README.md]: used "the" instead of "a" as it sounds a little better.

**Review by portends — APPROVED** [review]
Looks good to go :+1:

**Comment by nhanby46** [comment]
Great work :)

**Pull request merged**